composer.json

```diff
@@ -22,7 +22,7 @@
         }
     ],
     "require": {
-        "php": "^7.3"
+        "php": "^7.3 || ^8.0"
```

Comment:
Please use >=7.3 like recommended by composer

Reply:
I'm sorry, did you mean `"php": ">=7.3"`?
But it would be incorrect in case of sudden php9 release.

Reply:
> I'm sorry, did you mean `"php": ">=7.3"`?

Yes

> But it would be incorrect in case of sudden php9 release.

How do you know ? This is maybe compatible with php 9 too. ;)

Reply:
Ok. Done.

Reply:
Thanks